lib/rules/string-no-newline/README.md

```diff
@@ -70,3 +70,33 @@ a {
   font-family: "Times New Roman";
 }
 ```
+
+## Optional secondary options
+
+### `ignore: ["at-rule-preludes", "declaration-values"]`
+
+#### `"at-rule-preludes"`
+
+Ignore strings in at-rule preludes.
+
+The following patterns are _not_ considered problems:
+
+<!-- prettier-ignore -->
+```css
+@import url('foo
+.css');
```

Comment:
question

Sorry to ask this late in the process but why would you want to do that?

Reply:
We typically just say what the option does, e.g. "Ignore strings in xyz", as we don't assume what the user will do with that option. It may be for overlapping rules, or it maybe for some custom syntax thing.